private/append_common.m

```diff
@@ -131,6 +131,13 @@
             chansel = match_str(varargin{j}.label, oldlabel{j});
             data.(cfg.parameter{i})(:,chansel,chansel) = varargin{j}.(cfg.parameter{i})(:,chansel,chansel);
           end
+         
+        case {'chan_freq_time'}
```

Comment:
could you try and combine the two cases? i.e. rather than having

case {'chan_freq_time'}
  ...some-stuff
case {'chan' 'chan_time' 'chan_freq'}
  ...some-almost-identical-stuff

do something like

case {'chan' 'chan_freq' 'chan_time' 'chan_freq_time'}

...do the improved stuff by you (adding the semi-colons for the possibly present third dimension.